### PR TITLE
Display boost for KEEP-only pool

### PR DESCRIPTION
**Pull request by michalsmiarowski**
Closes: #2328 

Displays the reward boost for he KEEP ONLY pool:

image

### CODE REVIEW

File: solidity/dashboard/src/sagas/liquidity-rewards.js [hunk 1]
```diff
@@ -89,6 +89,11 @@ function* fetchLiquidityRewardsData(liquidityRewardPair, address) {
       )
     }
 
+    const rewardMultiplier = yield call(
```
Comment (r-czajkowski) [inline solidity/dashboard/src/sagas/liquidity-rewards.js:92]: We probably should update `rewardMultiplier` when the user has withdrawn or deposited more tokens.

File: solidity/dashboard/src/components/liquidity/RewardMultiplier.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        duration={1}
+        suffix={"x"}
+      />
+    </h2>
+  )
+}
+
+RewardMultiplier.TooltipContent = () => {
+  // TODO: content of the tooltip
+  return <>Reward multiplier tooltip</>
```
Comment (r-czajkowski) [inline solidity/dashboard/src/components/liquidity/RewardMultiplier.jsx:28]: I realize that we don't have wireframes for this but maybe: `Deposit liquidity tokens for 30 days to achieve a 3x reward multiplier.` + something about how we calculate it? I mean we take into account all stakes. What do you think?

**Review by r-czajkowski — CHANGES_REQUESTED** [review]

**Comment by michalsmiarowski** [comment]
Ready for another round of review @r-czajkowski

**Review by r-czajkowski — APPROVED** [review]
LGTM!

**Pull request merged**